params/protocol_params.go

```diff
@@ -146,13 +147,22 @@ func GetMaximumExtraDataSize() uint64 {
 	return BFTMaximumExtraDataSize
 }
 
+// CodeFormat is the version of the interpreter that smart contract uses
 type CodeFormat uint8
 
+// Supporting CodeFormat
```

Comment:
```suggestion
// Supporting CodeFormat
// CodeFormatLast should be equal or less than 16 because only the last 4 bits of CodeFormat are used for CodeInfo.
```